src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,20 @@
-export const UserInfo = () => {};
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    email,
+    name,
+  } = user;
+
+  return (
+    <a className="UserInfo" href={`mailto:${email}`}>
```

Comment:
Consider adding a target attribute to the anchor tag to open the email in a new tab. 🔗